### ISSUE TITLE
CopyQ v3.11.0 macOS: double click on entry from dropdown list does not insert

### ISSUE THREAD

**Issue by cmuerau** [hluk/CopyQ#1375]
**Describe the bug**
After updating from v3.10 to v3.11, no paste from dropdown menu by double clicking is possible. At the insertion point nothing gets inserted. Also self created keyboard commands do not insert anything, e.g. date and time just gets copied to clipboard but not inserted.

**To Reproduce**
1. Copy multiple items to the clipboard using Command-C keyboard shortcut
2. Click on CopyQ menu bar item to open dropdown menu
3. Click on any item which should be inserted
4. Nothing gets inserted at cursor

**Expected behavior**
Seleted content from list of clipboard items should be inserted at choosen insert point 

**Screenshots**
Not available/not suitable

**Version, OS and Environment**
CopyQ Clipboard Manager v3.11.0
Qt: 5.13.2
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: macOS Mojave (10.14)

**Additional context**
After discovering the bug, downgrade to 3.10.0 was done. When re-updating to 3.11.0 to reproduce error and create screenshots, the additional error "after double clicking entry in drop down, translucent window appears with clipboard content as title which cannot be closed" could not be reproduced.

**Comment by yuvmel** [comment]
Hi @cmuerau - at least on Catalina, the fix for me was to remove & add CopyQ again under Accessibility (macOS System Preferences, Security & Privacy, Privacy tab).

**Comment by hluk** [comment]
Can you see if [this FAQ entry](https://copyq.readthedocs.io/en/latest/faq.html#why-can-i-no-longer-paste-from-the-application-on-macos) helps?

**Comment by cmuerau** [comment]
Thanks @yuvmel and @hluk for pointing into the right direction.

Also in Mojava 10.14 it is possible to remove and add CopyQ again in the macOS System Preferences (for the German audience: Systemeinstellungen, Sicherheit, Datenschutz, Bedienungshilfen) to reactivate the expected behaviour.

And checking the FAQ link and especially #1030 helped a lot to better understand my issue.

What I didn't tell at first: just a few weeks ago I switched from traditional Drag-and-Drop to Homebrew. v3.11.0 was the first CopyQ version to be updated by Homebrew. In addition to this, I work with different accounts - admin account installs/updates via Homebrew and daily account just uses the apps. Cannot remember if macOS asked me to allow Accessibility ...

Might Homebrew and/or admin account have an impact on my version of this issue? 

Again thanks a lot, and I just documented it a little bit longer here for future reference - you newer know. ;-)
